observation_portal/requestgroups/serializers.py

```diff
@@ -849,6 +849,15 @@ def validate_requests(self, value):
         return value
 
 
+class CadenceRequestGroupSerializer(RequestGroupSerializer):
+    requests = import_string(settings.SERIALIZERS['requestgroups']['CadenceRequest'])(many=True)
+
+    # override the validate method from the RequestGroupSerializer and use the Cadence Request serializer to
+    # validate all of the Cadence Requests
+    def validate(self, data):
```

Comment:
The reason I overrode the validate method here is because the RequestGroupSerializer was going and looking for windows in the requests, which don't normally exist for a CadenceRequestGroup. 

What this effectively does is pass on the validation to the CadenceRequest serializer (which specifically excludes windows)

This gives us the desired documentation results and validates properly. Prior to this, we didn't have any upper-level CadenceRequestGroup Serializer - we just looped through each request and validated it using the CadenceRequest serializer. This effectively does the same thing, but also allows us to properly document the request body.